core/src/test/java/org/bitcoinj/wallet/WalletTest.java

```diff
@@ -3609,4 +3609,18 @@ public void oneTxTwoWallets() {
         assertEquals(txW1, txW2);
         assertNotSame(txW1, txW2);
     }
+
+    @Test
+    public void deprecatedMethods() {
```

Comment:
I named these tests `deprecatedMembers()` so we can check all members here, not only methods. But thanks for adding it!

Reply:
Fixed.  (Note some of the deprecated methods didn't have tests and neither do their replacements, I'll open up an issue for that.)